packages/matter-node.js/src/codec/BitmapSchema.ts

```diff
@@ -0,0 +1,168 @@
+/**
```

Comment:
This is a replacement for /packages/matter.js/src/schema/BitmapSchema.ts
I forked it to add more features but never merged it back.
Either:
1 _ do the overwrite in the MR
2 _ create another MR to do the overwrite now and I will approve both

Reply:
We decided to do "file merging" in a separate MR after the initial move. I will work on this once this one is in

Reply:
OK